src/app/services/auth.service.ts

```diff
@@ -1,16 +1,16 @@
 import { Injectable } from '@angular/core';
 import { AngularFireAuth } from '@angular/fire/auth';
-import { FirebaseService } from './firebase.service';
 import { switchMap, map, shareReplay } from 'rxjs/operators';
 import { from, of, Observable } from 'rxjs';
+import 'firebase/auth';
```

Comment:
Oh, I know I just approved this PR, but it might make sense to comment this import.

```suggestion

// Make sure the auth service is included in the main bundle
// (not lazy-loaded).
import 'firebase/auth';
```